aap_compose_dev.yaml

```diff
@@ -88,24 +91,45 @@ services:
     volumes:
       - "etc_pulp_certs:/etc/pulp/certs"
       - "var_lib_pulp:/var/lib/pulp"
+      - ".:/src/galaxy_ng"
     command: |
       bash -c "
-        if [[ ! -e /etc/pulp/certs/database_fields.symmetric.key ]] || [[ -s /etc/pulp/certs/database_fields.symmetric.key ]]; then
-          mkdir -p /etc/pulp/certs/;
-          echo 'check openssl and install ...';
-          rpm -q openssl || dnf -y install openssl;
-          echo 'generate key ...';
-          openssl rand -base64 32 > /etc/pulp/certs/database_fields.symmetric.key;
-          echo 'chown key ...';
-          chmod 640 /etc/pulp/certs/database_fields.symmetric.key;
-        else
-          echo 'symmetric key exists'
-        fi;
+        echo '#> STEP: Database Symmetric Key';
+        echo 'WARNING: Symmetric key is hardcoded for development only.';
+        echo 'DNmNdwgyZugTax9S64J0FITTr9IHPxbuoF1F1CGPr68=' > /etc/pulp/certs/database_fields.symmetric.key;
```

Comment:
Changed to use the same hardcoded key here, because it helps when database dumps are needed, I dumped the database and then it was challenging later to restore because the key was re-generated every time `down -v` + `up --build` was performed.

IT will be useful to persist postgres data folder to reuse.